### PR TITLE
build(*): replace yarn with pnpm

### PR DESCRIPTION
**Pull request by garrappachc**
Get rid of `yarn` in favor of `pnpm`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "mongoose": "8.0.3",
     "multer": "^1.4.5-lts.1",
     "nestjs-real-ip": "3.0.1",
+    "openapi3-ts": "^4.2.1",
```
Comment (c43721) [inline package.json:60]: ?

File: src/configuration/services/configuration.service.ts [hunk 2]
```diff
@@ -10,10 +10,11 @@ import {
 } from '../models/configuration-item';
 import { generateSchema } from '@anatine/zod-openapi';
 import { ZodType, ZodTypeAny } from 'zod';
+import { SchemaObject } from 'openapi3-ts/oas31';
 
 interface ConfigurationEntryDescription {
   key: string;
-  schema: string;
+  schema: SchemaObject;
```
Comment (c43721) [inline src/configuration/services/configuration.service.ts:17]: ???

**Review by c43721 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (garrappachc) [reply package.json:60]: yarn installed peer dependencies automatically, with pnpm I need to do it manually

File: src/configuration/services/configuration.service.ts (hunk 2 above)
Comment (garrappachc) [reply src/configuration/services/configuration.service.ts:17]: Yeah, for some reason...

Comment (c43721) [reply src/configuration/services/configuration.service.ts:17]: still makes no sense?

File: package.json (hunk 1 above)
Comment (c43721) [reply package.json:60]: openapi3-ts isn't used but in 1 spot?

Comment (garrappachc) [reply package.json:60]: Yeah, only in one spot...

File: src/configuration/services/configuration.service.ts (hunk 2 above)
Comment (garrappachc) [reply src/configuration/services/configuration.service.ts:17]: Apprarently, the `@anatine/zod-openapi` introduced a change in the return type of the `generateSchema()` function; yarn installed an older version (peer dependency), which returned a string; the newer version (installed by pnpm) returns `SchemaObject`, thus the change.

Comment (c43721) [reply src/configuration/services/configuration.service.ts:17]: where are we using @anatine/zod-openapi?

Comment (garrappachc) [reply src/configuration/services/configuration.service.ts:17]: In the Configuration module; it's used to describe configuration options' schema. It converts zod schema to the openapi standard.

Comment (c43721) [reply src/configuration/services/configuration.service.ts:17]: you know... this whole time I read openapi as openai thinking you had somehow gotten into ai unexpectedly. Damn.

**Review by c43721 — APPROVED** [review]
lgtm dr

**Pull request merged**